examples/cpp-simple/README.md

```diff
@@ -8,9 +8,50 @@ It will
 - Every 10 seconds, write a log saying "Hi! I'm a Game Server"
 - After 60 seconds, call `SDK::Shutdown()` to shut the server down.
 
+## Running
+
+If you want to run the example code without building the container yourself, you can start a single gameserver by running
+```
+kubectl create -f https://raw.githubusercontent.com/googleforgames/agones/master/examples/cpp-simple/gameserver.yaml
```

Comment:
Should this point to a release branch?

Reply:
Is that possible in the markdown files? I thought it only worked in the website directory?

Reply:
oooh. good point.

Maybe something  like:

```suggestion
kubectl create -f https://raw.githubusercontent.com/googleforgames/agones/<release-branch>/examples/cpp-simple/gameserver.yaml
```

With a note like: "Where `<release-branch>` is the Agones release you are working on, such as `release-0.11`"

WDYT?

???

Reply:
The nice thing about pointing to the master branch is that you can copy/paste and it should just work (obviously it wouldn't right now if you installed 0.11, but if we keep compatibility going forward it should work). I agree that it would be better to keep it tied to the release branch, which maybe argues for moving this over to the website....

For things in markdown in the git repository, I'm ok with them pointing at the master branch and not trying to keep them synchronized with releases. But I'll defer to you if you think it'd be better to instruct users towards a release branch.

Reply:
Yeah, I totally get the copy/paste impetus 👍

I've just had this conversation a lot:

Them: I tried the example thing, and it's broken
Me: Which version did you install?
Them: I grabbed the code from master
Me: I just updated the example with new features - it won't work against the current release.
Them: How am I supposed to know that?
Me: There is really no way, sorry. Use the release branch

So just trying to find away from that, since master will inevitably break at some point.

Maybe we should point them at release source zip downloads? Is that easier?

Reply:
I think you've convinced me that we should just move this over to the website so that it can be automatically linked to the right source. 

Both https://github.com/googleforgames/agones/tree/master/examples/allocator-service and https://github.com/googleforgames/agones/tree/master/examples/autoscaler-webhook do a good job of having a README but also having the main documentation that explains how to interact on the website where it's versioned.

Reply:
I think you've convinced me to move this over to the website 👍 

I like this a lot. I'd be happy with the website documentation to be a separate PR(s), unless you feel strongly you want to make the change in this PR?

Reply:
I've started working on it for this PR, so I'll finish that and send it your way today.

Reply:
I've pushed a new version that adds a tutorial to the website which should be published at about the same time as the 0.12 release.